spec/system/admin/order_spec.rb

```diff
@@ -951,10 +951,12 @@ def new_order_with_distribution(distributor, order_cycle)
           order.finalize! # ensure order has a payment to capture
           order.payments << create(:check_payment, order:, amount: order.total)
           order.payments.first.capture!
+          login_as_admin
```

Comment:
I could not figure out why the spec `@enterprise_user` had no permission to perform order ship action.